### DISCUSSION TITLE
PDF Code Annotation in C

### DISCUSSION THREAD

**Discussion by vfacta** [quarto-dev/quarto-cli#5771] [Q&A]
### Description

In a PDF output, code annotations within a code block are rendered as plain commentaries, whilst HTML outputs render them as a pretty numbered circle within the code block. 

in a code block such as:
````
```c
#include <stdio.h>

main() {

    int a, b, c; /* <1> */
        
    a = 2;
    b = 3;
    c = a + b;
        
    printf("%d + %d = %d\n", a, b, c);
}
```

1. More than one variable can be declared per line.
````

Even though the annotation below the code block appears as a numbered circled, the annotation in the code, `/* <1> */`, will appear as a plain commentary when rendered to PDF. 

As a feature request, I'd like to ask for a fix to code annotation for C code blocks in PDF output.

Thanks

**Comment by mcanouil** [comment]
I don't understand what you want and in which aspect it deviates from the expected behaviour.
Please, could you clarify?

What's wrong with the code annotation display? I don't see anything off.
<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/1527eb55-0cbb-4fc2-8d04-a5555df3ce75">

**Reply by vfacta** [reply]
For me, this is the rendered output:

![C code annotation](https://github.com/quarto-dev/quarto-cli/assets/120517270/b17eeb02-8fab-4be3-99b9-0df67c8a2621)

I notice that when i change the language for `python` in the `c` code block, the formatting works fine:

![C code annotation2](https://github.com/quarto-dev/quarto-cli/assets/120517270/11c906d9-e153-494f-a972-1a6b7df5170f)

**Reply by cscheid** [reply]
We recently fixed a bug like this - can you make sure you're running the latest version of quarto?

**Reply by vfacta** [reply]
Just installed the latest Quarto, 1.3.361, and the bug persists.

**Reply by cscheid** [reply]
I can't reproduce:

<img width="791" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/285675/afec6027-5531-4494-abab-131b730db13a">

Are you sure you're actually _running_ 1.3.361? You can test it via `quarto --version` or you can render an `html` version of the document and check the "generator" meta tag.

**Comment by mcanouil** [comment]
(Changing to Q&A.)
